internal/db/access_tokens.go

```diff
@@ -45,6 +45,7 @@ type AccessToken struct {
 	UserID int64 `xorm:"uid INDEX" gorm:"COLUMN:uid;INDEX"`
 	Name   string
 	Sha1   string `xorm:"UNIQUE VARCHAR(40)" gorm:"TYPE:VARCHAR(40);UNIQUE"`
+	SHA256 string `xorm:"VARCHAR(64)" gorm:"TYPE:VARCHAR(64);UNIQUE"`
```

Comment:
```suggestion
	SHA256 string `gorm:"COLUMN:sha256;TYPE:VARCHAR(64);UNIQUE"`
```

Not sure if the `COLUMN:sha256` is needed